fastlane/lib/fastlane/actions/testfairy.rb

```diff
@@ -76,7 +81,36 @@ def self.available_options
                                        short_option: '-g',
                                        env_name: "FL_TESTFAIRY_TESTERS_GROUPS",
                                        description: "Array of tester groups to be notified",
-                                       default_value: []) # the default value is an empty list
+                                       default_value: []), # the default value is an empty list
+          FastlaneCore::ConfigItem.new(key: :symbols_file,
+                                       env_name: "FL_TESTFAIRY_SYMBOLS_FILE",
+                                       description: "Symbols mapping file",
+                                       default_value: Actions.lane_context[SharedValues::DSYM_OUTPUT_PATH],
+                                       verify_block: proc do |value|
+                                         UI.user_error!("Couldn't find ipa file at path '#{value}'") unless File.exist?(value)
+                                       end),
+          FastlaneCore::ConfigItem.new(key: :notify,
+                                       env_name: "FL_TESTFAIRY_NOTIFY",
+                                       description: "Send email to testers",
+                                       default_value: 'on'),
+          FastlaneCore::ConfigItem.new(key: :auto_update,
+                                       env_name: "FL_TESTFAIRY_AUTO_UPDATE",
+                                       description: "Allows easy upgrade of all users to current version",
+                                       default_value: 'on'),
+          FastlaneCore::ConfigItem.new(key: :icon_watermark,
+                                       env_name: "FL_TESTFAIRY_ICON_WATERMARK",
+                                       description: "Add a small watermark to app icon",
+                                       default_value: 'on'),
+          FastlaneCore::ConfigItem.new(key: :metrics,
+                                       type: Array,
+                                       env_name: "FL_TESTFAIRY_METRICS",
+                                       description: "Array of metrics to record (cpu,memory,network,phone-signal,gps,battery,mic,wifi)",
```

Comment:
The docs are here: http://docs.testfairy.com/API/Upload_API.html

TBH I would rather suggest to rely on them for the validation. Otherwise it's yet another place that needs to be kept in sync.